packages/minifiers/minifiers-tests.js

```diff
@@ -13,9 +13,7 @@ Tinytest.add("minifiers - simple css minification", function (test) {
   t('p > *.my-class { color: #fff; }', 'p>.my-class{color:#fff}', 'removing optional whitespace around ">" in selector');
   t('p +  *.my-class { color: #fff; }', 'p+.my-class{color:#fff}', 'removing optional whitespace around "+" in selector');
   // XXX url parsing is difficult to support at the moment
-  t('a {\n\
-  font:12px \'Helvetica\',"Arial",\'Nautica\';\n\
-  background:url("/some/nice/picture.png");\n}',
+  t('a {\nfont:12px \'Helvetica\',"Arial",\'Nautica\';\nbackground:url("/some/nice/picture.png");\n}',
```

Comment:
This line is a result of us breaking the endline escaping thing... we should fix that and then unmake this change